packages/slate/test/interfaces/Text/decorations/in-a-row.js

```diff
@@ -0,0 +1,51 @@
+import { Text } from 'slate'
+
+export const input = [
+  {
```

Comment:
This is a great test, but I'm wondering whether the name is correct. Are these two adjacent decorations or are they overlapping? 

```
0  1  2  3  4
a [b (c] d) e
```
Where decoration one is [] and decoration two is ().

Am I picturing this correctly, or does one of the ends of the decoration not actually contain the character at the index?

Reply:
Oh and a follow up, if this test is indeed "overlapping" decorations, then should we have one for adjacent decorations also?

Reply:
Yeah sorry my english is not that good. This test is about adjacent decorations. I will change it.

Reply:
Done :)